src/components/CustomButton/CustomButton.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styles from "./CustomButton.module.css"
+
+const CustomButton = ({ text, onClick, customStyles }) => {
+  return (
```

Comment:
Bug: the button is not properly styled, it will not take entire width by default, please handle this case as well

Reply:
ok sure

Reply:
done